dll/win32/shell32/wine/shell32_main.c

```diff
@@ -568,28 +568,30 @@ DWORD_PTR WINAPI SHGetFileInfoW(LPCWSTR path,DWORD dwFileAttributes,
         else
         {
             if (dwFileAttributes & FILE_ATTRIBUTE_DIRECTORY)
-                strcatW (psfi->szTypeName, L"Folder");
+                LoadStringW(shell32_hInstance, IDS_DIRECTORY, psfi->szTypeName, 80);
```

Comment:
It is good practice to not hardcode the length of the buffer, instead of `80` replace it with `_countof(psfi->szTypeName)` to compute the count of elements of the string array. Same thing for the `StringCchPrintfW` call below.